src/wallet/rpc/util.cpp

```diff
@@ -123,6 +123,15 @@ std::string LabelFromValue(const UniValue& value)
     return label;
 }
 
+void PushWalletDescriptors(const CWallet& wallet, const CScript& script_pubkey, UniValue& entry)
+{
+    UniValue wallet_descs(UniValue::VARR);
+    for (const auto& desc: wallet.GetWalletDescriptors(script_pubkey)) {
+        wallet_descs.push_back(desc.descriptor->ToString());
+    }
+    entry.pushKV("wallet_descs", wallet_descs);
```

Comment:
Maybe call this `parent_descs`? The term `wallet_` is redundant for a wallet RPC.

The `listunspent` RPC already a field called `desc`. This has the limitation that it's not actually looking at the wallet descriptor.

In a followup PR we may want to add `descs` to get descriptor(s) for an individual address, rather than the full range. See #24003.

Reply:
Indeed `parent_descs` is a better name, thanks. Done.